OxfordBodleian/Juel-Jensen/BDLaethe33.xml

```diff
@@ -218,39 +258,88 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         
                         <handDesc>
                             <handNote xml:id="h1" script="Ethiopic">
-                           
+                                <seg type="script"/>
+                                <locus target="#1r"/>
+                                <seg type="ink">Black, red</seg>
+                                <date notBefore="1700" notAfter="1799">18th century</date>
+                                <seg type="rubrication">
+                                    The name of Giyorgis; the word <foreign xml:lang="gez">ሃሌ፡</foreign>; elements of the punctuation signs and numerals.
+                                </seg>
+                                <list type="abbreviations">    
+                                    <item>
+                                        <abbr>ሃሌ፡</abbr>, for <expan>ሃሌ፡ ሉያ፡</expan> (<locus target="#1r"/>)
+                                    </item>
+                                </list>
+                            </handNote>
+                            <handNote xml:id="h2" script="Ethiopic">
+                                <locus from="2r" to="130v"/>
+                                <seg type="script"/>
+                                <seg type="ink">Black, red</seg>
+                                <date notBefore="1700" notAfter="1799">18th century</date>
+                                <desc>
+                                    Small and occasionally irregular handwriting.
+                                </desc>
+                                <seg type="rubrication">
+                                    Divine names; number and titles of the psalms; name, number, and traditional interpretation of the Hebrew letter in Ps. 118; 
```

Comment:
If I remember correctly, we agreed on "Holy names" in the meeting?

Reply:
you're right! It was inherited from a previous description stage and left uncorrected.